crossdock/src/tchannel_server.js

```diff
@@ -35,9 +36,14 @@ export default class TChannelServer {
     this._helpers = new Helpers(this._tracer);
 
     let serverChannel = TChannel({ serviceName: 'node' });
+
     let tchannelThrift = TChannelThrift({
       channel: serverChannel,
-      entryPoint: crossdockSpecPath,
+      source:
+        ThriftData[crossdockSpecPath] ||
+        (() => {
+          throw new Error(`${crossdockSpecPath} not found`);
+        }),
```

Comment:
does this need trailing `()` in order for the function to be executed?

Reply:
This was inline debugging I forgot to remove. It would fail anyway if any invalid path was used (and we don't do any kind of entry-point sanity check anywhere else) so definitely no needed.